### PR TITLE
Example

### PR DESCRIPTION
**Pull request by jgarciab**
Improved examples and README

### CODE REVIEW

**Comment by jgarciab** [comment]
Please @modhurita review, and add yourself to the citation, once we have a new version we can put it in Zenodo and add your name there too

File: README.md [hunk 1]
```diff
@@ -32,147 +35,105 @@ a new line, e.g.:
...
+See [example notebook](examples/example_collect_all_artworks.ipynb).
 
-# Get links for all artists, as a list. The links are also saved in a file.
-artist_urls = get_artist_links(executable_path='geckodriver', min_wait_time=1, output_file='artist_links.txt')
-```
+The final structure of the results will be
+- data
+  - artist_links.txt (All artists, with one url per line) 
+  - Artist_1
+    - description.txt (Description of artist, from wikimedia)
```
Comment (modhurita) [inline README.md:96]: Should be wikidata instead of wikimedia.

File: README.md [hunk 2]
```diff
@@ -32,147 +35,105 @@ a new line, e.g.:
...
+    - metadata.json (Metadata of arist, from wikimedia)
```
Comment (modhurita) [inline README.md:97]: Same here, should be wikidata.

File: README.md [hunk 3]
```diff
@@ -32,147 +35,105 @@ a new line, e.g.:
...
 
-# Save the image to some local file
-scraper.save_image("wiki.jpg")
+## 3. Downloading art from Google Arts & Culture
 
-# Release resources
-scraper.close()
-```
+To download data from GoogleArt it is necessary to install 
+[Firefox](https://www.mozilla.org/en-US/firefox/new/) and `geckodriver`. Geckodriver is installed automatically when you run the code for the first time.
```
[the remark this answers is not in the document]
Comment (modhurita) [reply README.md:65]: Never mind, I see that it is now done automatically: https://github.com/sodascience/artscraper/blob/e0482ad2981bce379075690821f114068ff688d8/artscraper/find_artists.py#L28

Comment (modhurita) [inline README.md:65]: Is it not confusing to first say that geckodriver needs to be installed, and then say that it is installed automatically? Maybe it is better to leave out any mention of geckodriver?

File: examples/example_collect_all_artworks.ipynb [hunk 4]
```diff
@@ -42,291 +50,161 @@
...
+    "        for url in artwork_links:\n",
+    "            print(f'artwork URL: {url}')\n",
+    "            retry(subscraper.save_artwork_information, max_retries, min_wait_time, url)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "54afc420",
+   "execution_count": 5,
```
Comment (modhurita) [inline examples/example_collect_all_artworks.ipynb:150]: Cell 5 is empty and needs to be removed.

**Comment by modhurita** [comment]
Hi @jgarciab : 

Thanks for your work on improving this repository!

I have provided some comments above, and changed some things myself online. I hope they look okay to you.

I could run the Google Arts & Culture parts of the example notebooks, but not the WikiArt ones. I obtained API keys by creating a WikiArt account, and placed them in the examples directory, as per the instructions in the README. However, the very first example cell didn't execute successfully. This is the error I got:

```
FileNotFoundError                         Traceback (most recent call last)
~/ResearchEngineering/artscraper/artscraper/wikiart.py in __init__(self, output_dir, skip_existing, min_wait, timeout)
     23         try:
---> 24             with open(".wiki_session", "r", encoding="utf-8") as f:
     25                 self.session_key = f.read()

FileNotFoundError: [Errno 2] No such file or directory: '.wiki_session'

During handling of the above exception, another exception occurred:

KeyError                                  Traceback (most recent call last)
/tmp/ipykernel_13612/1175252453.py in <module>
      3 art_url = "https://www.wikiart.org/en/edvard-munch/anxiety-1894"
      4 
----> 5 with WikiArtScraper() as scraper:
      6     scraper.load_link(art_url)
      7     metadata = scraper.get_metadata()

~/ResearchEngineering/artscraper/artscraper/wikiart.py in __init__(self, output_dir, skip_existing, min_wait, timeout)
     25                 self.session_key = f.read()
     26         except FileNotFoundError:
---> 27             self._new_session()
     28             with open(".wiki_session", "w", encoding="utf-8") as f:
     29                 f.write(self.session_key)

~/ResearchEngineering/artscraper/artscraper/wikiart.py in _new_session(self)
     66                                 },
     67                                 timeout=self.timeout)
---> 68         self.session_key = json.loads(response.text)["SessionKey"]
     69         self.last_request = time.time()
     70 

KeyError: 'SessionKey'
```

Finally, at which position in the names list in the citation should I add my name?

Thanks,
Modhurita

File: examples/example_collect_all_artworks.ipynb (hunk 4 above)
Comment (jgarciab) [reply examples/example_collect_all_artworks.ipynb:150]: Yes please

**Comment by jgarciab** [comment]
Hi Modhurita, I only tried with the interactive version, could you try to figure it out? That was Raoul's part, if you don't understand it how it works maybe you could ask him.

You can add your name in second place in the citation if you are okay with that.

**Comment by modhurita** [comment]
Hi @jgarciab : 

The WikiArt part seems to work now - not sure why I got that error earlier. I have made the other changes. I approve the pull request; you can now merge this branch into main now.

**Pull request merged**